x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/processors_tree/processors_tree.scss

```diff
@@ -1,74 +1,63 @@
 @import '@elastic/eui/src/global_styling/variables/size';
 
 .pipelineProcessorsEditor__tree {
-
   &__container {
     background-color: $euiColorLightestShade;
     padding: $euiSizeS;
   }
 
   &__dropZoneContainer {
+    position: relative;
     margin: 2px;
     visibility: hidden;
-    border: 2px dashed $euiColorLightShade;
-    height: 12px;
-    border-radius: 2px;
-
-    transition: border .5s;
+    background-color: transparent;
+    height: 2px;
 
-    &--active {
+    &--visible {
       &:hover {
-        border: 2px dashed $euiColorPrimary;
+        background-color: $euiColorPrimary;
       }
       visibility: visible;
     }
+
+    &--unavailable {
+      &:hover {
+        background-color: $euiColorMediumShade;
+      }
+    }
+
+    &__toolTip {
+      pointer-events: none;
+    }
   }
 
   &__dropZoneButton {
-    height: 8px;
+    position: absolute;
+    padding: 0;
+    height: 60px;
```

Comment:
Might be worth extracting this into a variable and then using math to calculate the `margin-top` value. For example:

```scss
$dropZoneButtonHeight: 60px;
$dropZoneButtonOffsetY: $dropZoneButtonHeight * -0.5;
```